bindings/python/tokenizers/implementations/base_tokenizer.py

```diff
@@ -13,6 +13,18 @@ def __repr__(self):
             self._tokenizer.get_vocab_size(),
             ', '.join(k + '=' + str(v) for k, v in self._parameters.items()))
 
+    def get_vocab_size(self, with_added_tokens: bool = True):
+        """ Return the size of vocabulary, with or without added tokens.
+
+        Args:
+            with_added_tokens: (`optional`) bool:
+                Whether to count in added special tokens or not
+
+        Returns:
+            Size of vocabulary
+        """
+        return self._tokenizer.get_vocab_size(with_added_tokens)
```

Comment:
Thank you for this PR @kdexd! If I remember correctly `with_added_tokens` should be a named argument and so I don't think it works if provided as a positional argument.

Reply:
Thanks for pointing out @n1t0 ! I hadn't added this option in my local patched version, glad it didn't slip by.